pkgs/by-name/go/gotenberg/package.nix

```
@@ -0,0 +1,100 @@
+{
+  lib,
+  buildGoModule,
+  chromium,
+  fetchFromGitHub,
+  libreoffice,
+  makeWrapper,
+  pdftk,
+  qpdf,
+  unoconv,
+  mktemp,
+  makeFontsConf,
+  liberation_ttf_v2,
+  jre17_minimal,
+  exiftool,
+  python311,
+  nix-update-script,
+}:
+let
+  fontsConf = makeFontsConf { fontDirectories = [ liberation_ttf_v2 ]; };
+  jre' = jre17_minimal.override {
+    modules = [
+      "java.base"
+      "java.desktop"
+      "java.logging"
+      "java.sql"
+    ];
+  };
+  libreoffice' = "${libreoffice}/lib/libreoffice/program/soffice.bin";
+  unoconv' = unoconv.override { python3 = python311; };
+in
+buildGoModule rec {
+  pname = "gotenberg";
+  version = "8.8.0";
+
+  src = fetchFromGitHub {
+    owner = "gotenberg";
+    repo = "gotenberg";
+    rev = "refs/tags/v${version}";
+    hash = "sha256-OHvtdUFZYm+I8DN8lAdlNAIRLWNRC/GbVEueRfyrwDA=";
+  };
+
+  vendorHash = "sha256-w9Q3hK8d5NwDYp8kydrWrlBlOmMyDqDixdv3z79yhK8=";
+
+  patchPhase = ''
+    find ./pkg -name '*_test.go' -exec sed -i -e 's#/tests#${src}#g' {} \;
+  '';
+
+  nativeBuildInputs = [ makeWrapper ];
```

```suggestion
  nativeBuildInputs = [ makeBinaryWrapper ];
```

we don't use variable expansion or anything here, so i think we should be fine with a (faster) binary wrapper